inspect/inspect.go

```diff
@@ -76,9 +81,8 @@ func NewFromConfig(cfg *config.Config) (*Inspector, error) {
 	if err != nil {
 		return nil, err
 	}
-	logger := log.MustNewDefaultLogger(log.LogFormatPlain, log.LogLevelInfo, false)
 	ss := state.NewStore(sDB)
-	return New(cfg.RPC, bs, ss, sinks, logger), nil
+	return makeInspector(cfg.RPC, bs, ss, sinks, logger), nil
```

Comment:
I have a slight preference against doing this. It means that we can't easily test our public API because we can only return what can be constructed from `NewFromConfig` and this `makeInspector` function is not necessarily going to continually be used in the same way. I'd slightly prefer we just move the main `Inspector` constructor into an `internal` package so that we can test it in the same way as it is tested here, as a 'public' API of an internal package but that can only be consumed by this package.

Reply:
Is `inspect` meaningfully useful as a part of the public API? (I'd argue not, but not too strongly, if there are use cases that I've missed.) The package of course has a public API but that's probably only really of use to internal users (because we dont' want the low level API's to the storage and state machine to become part of the public API of tendermint).

The other option, I think, is that we revert the last commit on this branch and move the whole thing to `internal/inspect`,

Reply:
I don't see anything particularly at issue with internalizing the whole thing since it's, in effect, going to be entirely internal to Tendermint.

Reply:
on it!